### ISSUE TITLE
appc android issue 2373

### ISSUE THREAD

**Issue by Lenndev** [caffeinalab/ti.goosh#69]
Hi,

Untill a few minutes ago, i had an open issue, asking assistance for the implementation.

I did get a response after posting my tiapp.xml.. i changed some values i.e. user ids and my app package name and project id. I did not do this last change consistent, resulting in two different values for app id and package name id. 

in the complete tiapp.xml i do have the same value for these properties ( ofcourse) but the issue is now closed, making me to create this issue ticket... i don't say it is an issue in the module, it might indeed be an app related problem... but i just ask for assistance here since the problem is "related" push notification handling using this module.. i just wonder if i am doing something wrong... i can't find a concrete solution to the problem (android issue 2373 (or something like that)  on the rest of the internet... they are all pretty vague... 

Let me rephrase my question... do i need to create an intent to start my app if i open a notification with the app in background? or is that all handled in this module...

**Comment by progress44** [comment]
Hi,
You can reopen the issue eventually. 
The module handles everything on it's own, no intent needs to be defined.
